include/global.h

```diff
@@ -87,6 +87,8 @@
 // Converts a Q16.16 fixed-point format number to a regular integer
 #define Q_16_16_TO_INT(n) ((int)((n) >> 0x10))
 
+#define Q_24_8_MULTIPLY(intVal, floatVal) Q_24_8_TO_INT((intVal)*Q_24_8(floatVal))
```

Comment:
Idk if you need to make this a macro, feels like this isn't something they would have written

Reply:
You think so?
I feel like it's more likely they did it, than writing out the contents of this macro every time they multiply like that.

Reply:
IMO it's a tradeoff between readability, transparency, and shared code. In this case:

- The macro makes the function more readable
- The macro makes it less transparent what's going on
- The macro only removes a few characters of code